edb/server/protocol/protocol.pyx

```diff
@@ -523,6 +524,8 @@ cdef class HttpProtocol:
                 # Check if this is a request to a registered extension
                 if extname == 'edgeql':
                     extname = 'edgeql_http'
+                if extname == 'ext':
+                    extname = path_parts[3]
```

Comment:
A comment is appreciated here as the logic can be confusing - we want to support both the old style API `/db/{dbname}/{ext_name}` and the new style API `/db/{dbname}/ext/{ext_name}` here I think?

Also the later `args = path_parts[3:]` is inconsistent, suggested:

```suggestion
                if extname == 'ext':
                    extname = path_parts[3]
                    args = path_parts[4:]
                else:
                    args = path_parts[3:]
```

So that you don't need to use `args[1:]`.